src/util-decode-der-get.c

```diff
@@ -205,6 +206,75 @@ int Asn1DerGetIssuerDN(const Asn1Generic *cert, char *buffer, uint32_t length,
     return rc;
 }
 
+int Ans1DerGetValidityParseTimestamp(char *dst, uint32_t dst_len,
+        char *src, uint32_t src_len)
+{
+    int rc = -1;
+    struct timeval ts;
+    struct tm tm;
+    time_t epoch;
+
+    if (strptime(src, "%d%m%y%H%M%SZ", &tm) == NULL)
```

Comment:
This does not take into consideration if the year is 19YY or 20YY. It also uses local time instead of UTC, which would work poorly, since the validity dates are using UTC.